src/utils/azureClients.ts

```diff
@@ -34,3 +35,7 @@ export function createContainerRegistryClient(context: AzExtClientContext, regis
 export async function createOperationalInsightsManagementClient(context: AzExtClientContext): Promise<OperationalInsightsManagementClient> {
     return createAzureClient(context, (await import('@azure/arm-operationalinsights')).OperationalInsightsManagementClient);
 }
+
+export async function createAuthorizationManagementClient(context: AzExtClientContext): Promise<AuthorizationManagementClient> {
+    return createAzureClient(context, (await import('@azure/arm-authorization')).AuthorizationManagementClient);
```

Comment:
Might make more sense to import this from the azureutils package. It works for custom clouds as well:

```
export async function createAuthorizationManagementClient(context: InternalAzExtClientContext): Promise<AuthorizationManagementClient> {
    if (parseClientContext(context).isCustomCloud) {
        return <AuthorizationManagementClient><unknown>createAzureClient(context, (await import('@azure/arm-authorization-profile-2020-09-01-hybrid')).AuthorizationManagementClient);
    } else {
        return createAzureClient(context, (await import('@azure/arm-authorization')).AuthorizationManagementClient);
    }
}
```